### PR TITLE
c64_cass.xml: Added 10 items (9 working, 1 not working)

### PR DESCRIPTION
**Pull request by scoomby**
New working software list additions
---------------------------------------
Time Tunnel (U.S. Gold) [C64 Ultimate Tape Archive V2.0]
Tintin on the Moon (Infogrames) [C64 Ultimate Tape Archive V2.0]
Token of Ghall (Interceptor Software) [C64 Ultimate Tape Archive V2.0]
Tom & Jerry (Magic Bytes) [C64 Ultimate Tape Archive V2.0]
Tom (Kingsoft) [C64 Ultimate Tape Archive V2.0]
Top Gun (Ocean) [C64 Ultimate Tape Archive V2.0]
The Train: Escape to Normandy (Electronic Arts) [C64 Ultimate Tape Archive V2.0]
The Transformers: Battle to Save the Earth (Activision) [C64 Ultimate Tape Archive V2.0]
Transylvanian Tower (Richard Shepherd Software) [C64 Ultimate Tape Archive V2.0]

New NOT_WORKING software list additions
---------------------------------------
Trailblazer (Gremlin Graphics) [C64 Ultimate Tape Archive V2.0]

### CODE REVIEW

File: hash/c64_cass.xml [hunk 1]
```diff
@@ -15382,6 +15382,80 @@ Some are based on the C64 Tape Archive at https://archive.org/download/c64tapes/
...
 	</software>
 
+	<software name="timetunn">
+		<description>Time Tunnel</description>
+		<year>1986</year>
+		<publisher>U.S. Gold</publisher>
+
+		<part name="cass1" interface="cbm_cass">
+			<dataarea name="cass" size="455643">
+				<rom name="Time_Tunnel.tap.tap" size="455643" crc="40d446aa" sha1="b03646587ca1aca5dc242395101c5b5b7bb107bb"/>
```
Comment (0kmg) [inline hash/c64_cass.xml:15392]: .tap.tap? Is this kosher?

File: hash/c64_cass.xml [hunk 2]
```diff
@@ -15382,6 +15382,80 @@ Some are based on the C64 Tape Archive at https://archive.org/download/c64tapes/
...
+			</dataarea>
+		</part>
+
+		<part name="cass2" interface="cbm_cass">
+			<dataarea name="cass" size="455643">
+				<rom name="Time_Tunnel.tap_a1.tap" size="455643" crc="d4805b29" sha1="6050ce7a853e4b042f3c787d4574d584d171d65e"/>
```
Comment (0kmg) [inline hash/c64_cass.xml:15398]: Not sure if this is in A1 condition either...

File: hash/c64_cass.xml (hunk 1 above)
Comment (scoomby) [reply hash/c64_cass.xml:15392]: It isn't kosher!  Needs a fix :-)

File: hash/c64_cass.xml (hunk 2 above)
Comment (scoomby) [reply hash/c64_cass.xml:15398]: Nowhere near A1 condition!  Thanks for spotting both typo's :-)

**Pull request merged**